src/components/WalletsList/WalletsList.jsx

```diff
@@ -0,0 +1,62 @@
+import { Box, Flex, Text, useColorModeValue, VStack } from '@chakra-ui/react';
+
+import { PiggyBankIcon } from '@/assets';
+
+import { AddWalletModal } from '../AddWalletModal';
+import { WalletCard } from '../WalletCard';
+
+export const WalletsList = () => {
+  const iconsThemeColor = useColorModeValue('teal.900', 'orange.300');
+  const bgSection = useColorModeValue('orange.200', 'teal.700');
+
+  return (
+    <>
+      <Box
+        as="section"
+        w="100%"
+        maxW="container.xl"
+        p={0}
+        bg={bgSection}
+        borderRadius={35}
+        shadow="lg"
+      >
+        <Flex justify="center">
+          <VStack
+            w="50vw"
+            p={8}
+            spacing={8}
+            alignItems="flex-start"
+            justify="center"
+            // bg="gray.400"
+          >
+            <Text as="h2" fontWeight="bold">
+              Total Balance:{' '}
+            </Text>
+            <Text>Choose your wallet</Text>
+            <AddWalletModal />
+          </VStack>
+          <VStack
+            w="100vw"
+            p={8}
+            spacing={8}
+            justify="center"
+            // bg="gray.300"
+          >
+            <WalletCard />
```

Comment:
WalletCard component has too big default width and height => it will be difficult to render this component in carousel (according to requirements).
Or maybe I try to implement this in a wrong way.
As I said - suggestions are welcome!

Reply:
Maybe it is better to use some adaptive values like % instead of px inside WalletCard component?

Reply:
Good question for our mentors 🙂